.NET/Microsoft.Recognizers.Text.Number/Parsers/BaseNumberParser.cs

```diff
@@ -696,21 +696,21 @@ private bool SkipNonDecimalSeparator(char ch, int distance)
             return ch == Config.NonDecimalSeparatorChar && !(distance <= decimalLength && isMultiDecimalSeparatorCulture);
         }
 
-        private List<string> GetMatches(string input)
-        {
-            var successMatch = TextNumberRegex.Match(input);
-            var matchStrs = new List<string>();
-
-            // Store all match str.
-            while (successMatch.Success)
-            {
-                var matchStr = successMatch.Groups[0].Value;
-                matchStrs.Add(matchStr);
-                successMatch = successMatch.NextMatch();
-            }
-
-            return matchStrs;
-        }
+        // private List<string> GetMatches(string input)

```

Comment:
This whole method was commented, shouldn't it be removed instead?

Reply:
It should, I just forgot about it, but I removed it already, thank you!